### ISSUE TITLE
Congress Crawler is broken

### ISSUE THREAD

**Issue by 7onn** [7onn/politicians-contacts#1]
**What happened**?
I noticed that the congress csv file is empty.

[Logs](https://github.com/7onn/politicians-contacts/actions/runs/3310931279/jobs/5465759697) from GitHub actions:
```
Traceback (most recent call last):
  File "/home/runner/work/politicians-contacts/politicians-contacts/./scripts/congress_crawler.py", line 77, in run
    total = self.get_total_congress(legislature)
  File "/home/runner/work/politicians-contacts/politicians-contacts/./scripts/congress_crawler.py", line 64, in get_total_congress
    total = re.findall(r"\d{3,}", pfound)[0]
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/re.py", line 241, in findall
    return _compile(pattern, flags).findall(string)
TypeError: expected string or bytes-like object
2022-[10](https://github.com/7onn/politicians-contacts/actions/runs/3310931279/jobs/5465759697#step:6:11)-24 07:52:21,263 INFO:Congress crawler exited
```

**Comment by 7onn** [comment]
The congress crawler attempted to reach `https://www.camara.leg.br/deputados/quem-sao/resultado?search=&partido=&uf=&sexo=&legislatura=57` but encountered no congresspeople in this page. I believe this is because elections took place and the next legislature have been created but not yet filled.

I'll try to make a fix decreasing the legislature number by one when in a month between October and December.

The last successful [run](https://github.com/7onn/politicians-contacts/actions/runs/3172358927/jobs/5166748170) happened 22 days ago (2022-10-03). The first [error](https://github.com/7onn/politicians-contacts/actions/runs/3217635352/jobs/5260840772) 15 days ago (2022-10-10).

**Comment by 7onn** [comment]
Actually, it makes no sense to fetch the quorum of a past legislature.

Instead, I came up with preventing GitHub actions from commit the file if the CSV don't change.
https://github.com/7onn/politicians-contacts/commit/606cd3f28dc149865e35b975d6726dedd5596bba

And in the script side, I improved error messages and added conditionals to prevent the CSV from being written when the quorum is empty.
https://github.com/7onn/politicians-contacts/commit/606cd3f28dc149865e35b975d6726dedd5596bba
